packages/font/src/google/fetch-font-file.ts

```diff
@@ -1,6 +1,7 @@
 // @ts-ignore
 import fetch from 'next/dist/compiled/node-fetch'
 import { getProxyAgent } from './get-proxy-agent'
+import { retry } from './retry'
```

Comment:
Let's use [`async-retry`](https://www.npmjs.com/package/async-retry) instead of the self-baked solution.